tests/main/services-refresh-mode/task.yaml

```diff
@@ -1,4 +1,10 @@
-summary: "Check that refresh-modes works"
+summary: Check that refresh-modes works
+
+details: |
+    Snapd maintains the process id when a service snap is refreshed.
```

Comment:
I think this is somewhat misleading. All the test checks is that a snap can designate a service that is _not touched_ by the refresh process. It's simply _not_ stopped or started again.

Reply:
you are right, just checked this service is configure with "refresh-mode: endure"